src/tests/Calculator.test.js

```diff
@@ -0,0 +1,8 @@
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Please use React testing library to create some integration tests that simulate user interaction

Reply:
Thanks for that suggestion! I'll implement user interaction checks in the Calculator page right away!